### PR TITLE
Prepare End User Structure

### PR DESCRIPTION
**Pull request by ahmednasserzaza**

### CODE REVIEW

File: client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import com.seiko.imageloader.LocalImageLoader
+import util.generateImageLoader
+import util.getPlatformContext
+
+
+private val localDrawableResources = staticCompositionLocalOf { DrawableResources() }
+private val localStringResources = staticCompositionLocalOf { StringResources() }
+
+@Composable
+fun BpRestaurantTheme(
```
Comment (mosayed01) [inline client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt:19]: rename this to match end user app

Comment (ahmednasserzaza) [reply client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt:19]: Oh , the dark side of duplication 😆

File: client_end_user/shared/src/commonMain/kotlin/util/ImagePicker.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.ImageBitmap
+
+expect class ImagePickerFactory(context: PlatformContext) {
+
```
Comment (NadaFeteiha) [inline client_end_user/shared/src/commonMain/kotlin/util/ImagePicker.kt:7]: No need for an image picker here, I don't see any uploading image in the design we can add it later if they need it.

File: client_end_user/shared/src/commonMain/kotlin/resources/DrawableResources.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package resources
+
+data class DrawableResources(
+    val bpIcon: String = "ic_beep_beep_icon.xml",
```
Comment (NadaFeteiha) [inline client_end_user/shared/src/commonMain/kotlin/resources/DrawableResources.kt:4]: I think you need to delete all this resources just set one for test

File: client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    useDarkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit,
+) {
+    val drawableResources = if (useDarkTheme) BpDrawableDarkResources else DrawableResources()
+    val context = getPlatformContext()
+
+    CompositionLocalProvider(
+        localDrawableResources provides drawableResources,
+        localStringResources provides StringResources(),
+        LocalImageLoader provides remember { generateImageLoader(context) }
```
Comment (NadaFeteiha) [inline client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt:28]: as Barq said we need to delete LocalImageLoader from here

**Review by NadaFeteiha — CHANGES_REQUESTED** [review]

**Review by ay0o5h — APPROVED** [review]

File: client_end_user/shared/src/commonMain/kotlin/resources/DrawableResources.kt (hunk 3 above)
Comment (ahmednasserzaza) [reply client_end_user/shared/src/commonMain/kotlin/resources/DrawableResources.kt:4]: Done

File: client_end_user/shared/src/commonMain/kotlin/util/ImagePicker.kt (hunk 2 above)
Comment (ahmednasserzaza) [reply client_end_user/shared/src/commonMain/kotlin/util/ImagePicker.kt]: Done

File: client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt (hunk 4 above)
Comment (ahmednasserzaza) [reply client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt]: Done

**Review by NadaFeteiha — APPROVED** [review]

**Pull request merged**